src/core/reference/src/op/non_max_suppression.cpp

```diff
@@ -366,35 +366,34 @@ void non_max_suppression(const float* boxes_data,
     }
 }
 
-OPENVINO_SUPPRESS_DEPRECATED_START
-void nms_postprocessing(const HostTensorVector& outputs,
+void nms_postprocessing(ov::TensorVector& outputs,
                         const ngraph::element::Type output_type,
                         const std::vector<int64_t>& selected_indices,
                         const std::vector<float>& selected_scores,
                         int64_t valid_outputs,
                         const ngraph::element::Type selected_scores_type) {
-    outputs[0]->set_element_type(output_type);
-    outputs[0]->set_shape(Shape{static_cast<size_t>(valid_outputs), 3});
+    // outputs[0].set_element_type(output_type);
+    outputs[0].set_shape(Shape{static_cast<size_t>(valid_outputs), 3});
 
     size_t num_of_outputs = outputs.size();
 
     if (num_of_outputs >= 2) {
-        outputs[1]->set_element_type(selected_scores_type);
-        outputs[1]->set_shape(Shape{static_cast<size_t>(valid_outputs), 3});
+        // outputs[1].set_element_type(selected_scores_type);
+        outputs[1].set_shape(Shape{static_cast<size_t>(valid_outputs), 3});
     }
 
     if (num_of_outputs >= 3) {
-        outputs[2]->set_element_type(output_type);
-        outputs[2]->set_shape(Shape{1});
+        // outputs[2].set_element_type(output_type);
```

Comment:
remove commented code?

Reply:
Fixed

Reply:
we have several other similar places in this file

Reply:
Removed all comments in backend ops